### ISSUE TITLE
Wrong swipe directions

### ISSUE THREAD

**Issue by grimalschi** [planecore/MuseBar#5]
Hey!

Now we have:

> Flick left/right to go to the previous/next song.

Should be opposite behaviour. When you swipe left it means "next". But I am getting previous song.

**Comment by planecore** [comment]
Why should it be the opposite behaviour?

<img width="150" alt="image" src="https://user-images.githubusercontent.com/12946462/84235991-8f8d3580-aaff-11ea-84b2-6dc7e41c2def.png">

On music players the left button is for the previous song and the right button for the next song, so it makes sense to swipe to the left for the previous song and swipe to the right for the next song.

**Comment by planecore** [comment]
Because of no activity, I close the issue.

**Comment by grimalschi** [comment]
This is dark day for UX

**Comment by Starbix** [comment]
I agree with @grimalschi, I find it more natural to swipe to the left to skip to the next song. Why not offer a setting to decide between the directions? Or would that mean too much new code?

**Comment by rpendleton** [comment]
For what it's worth, I've been using a really old version of Muse Bar and just updated to the latest version from this repo, and after updating, I noticed the swipe direction is wrong now.

At first I thought I was just losing my mind or that the app had a regression or something, because I kept restarting and going back to previous tracks instead of skipping to the next track, and it's been surprisingly annoying. It turns out it's just a change in behavior in this fork.

I came here to report the exact same issue and was surprised to find that it's been like this for years apparently. I was even more surprised to find out that it's considered expected behavior, since every other track-changing gesture I've seen on iOS uses natural scroll directions.

After looking at the Git history, it turns out the app actually used to use natural scroll directions (and there's even a comment left over in the code saying that it uses natural scroll directions), but it seems the directions were changed to non-natural as part of this commit:

https://github.com/planecore/MuseBar/commit/3aa7f535579d19c52698cc840b69463e3eac1206#diff-18d48be6f1dd12b0b899fcfd8792fb053dad3666af7690f34366b642e85be858R507-R510

It's not ideal, but for those that prefer the original natural scrolling direction, it's simple enough to just swap the `<` to `>` in `WindowController.controlStripButtonPanGestureHandler` to get the original behavior back:

```patch
diff --git a/Muse/Sources/Views/WindowController.swift b/Muse/Sources/Views/WindowController.swift
index ff136ce..7ae5130 100644
--- a/Muse/Sources/Views/WindowController.swift
+++ b/Muse/Sources/Views/WindowController.swift
@@ -444,7 +444,7 @@ class WindowController: NSWindowController, NSWindowDelegate, SliderDelegate {
         switch recognizer.state {
         case .began:
             // Reverse translation check (natural scroll)
-            if recognizer.translation(in: controlStripButton).x < 0 {
+            if recognizer.translation(in: controlStripButton).x > 0 {
                 helper.previousTrack()
             } else {
                 helper.nextTrack()
```

Long-term, it might be nice to have some sort of preference as others have suggested. For now though, just flipping the sign and building the app myself has worked for me.

(It's also interesting to note that the same commit that changed the swipe direction also removed preferences... So I'm not sure where a setting for this would actually belong. Perhaps at the very least, it could be something that can be changed using `defaults` on command line.)
